lib/screen/settings/crypto/wallet_detail/linked_wallet_detail_page.dart

```diff
@@ -92,11 +92,24 @@ class _LinkedWalletDetailPageState extends State<LinkedWalletDetailPage>
 
   void _callBloc() {
     final cryptoType = widget.payload.type;
-    context
-        .read<WalletDetailBloc>()
-        .add(WalletDetailBalanceEvent(cryptoType, _address));
-    if (cryptoType == CryptoType.ETH) {
-      context.read<USDCBloc>().add(GetUSDCBalanceWithAddressEvent(_address));
+
+    switch (cryptoType) {
+      case CryptoType.ETH:
+        // for erc20
+        context.read<USDCBloc>().add(GetUSDCBalanceWithAddressEvent(_address));
+        continue allCrypto;
```

Comment:
simply remove the continue, the result will be the same (just don't `break`, it will fallthrough the next case)

Reply:
tried it, but not worked 
https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=switch_case_completes_normally#switch_case_completes_normally

Reply:
then I think we can simply write full code for every case, It can be duplicated code a little bit, but it will be easier to read.

Reply:
ok